stan/math/gpu/add_opencl.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef STAN_MATH_GPU_ADD_OPENCL_HPP
```

Comment:
This file should just be named `add` since the function’s name is add. There’s no need for appending `opencl` to the filename. It’s already sitting in the gpu directory.